src/app/view/view-controls/view-controls.component.scss

```diff
@@ -1,5 +1,9 @@
 @import "../../shared/variables";
 
+.nowrap{
+  white-space: nowrap;
+}
```

Comment:
Nice solution! However, Bootstrap already provides [.text-nowrap](https://getbootstrap.com/docs/4.0/utilities/text/#text-wrapping-and-overflow) class so there is no need for implementing it again.